examples/deploy_tasks.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: pipeline.knative.dev/v1beta1
+kind: Task
+metadata:
+  name: deploy-with-helm
+  namespace: default
+spec:
+    inputs:
+        sources:
+            - name: workspace
+        params:
+            - name: pathToHelmCharts
+              type: string
+            - name: helmArgs
+              type: string
+            - name: image
+              type: string
+        cluster:
+            - name: clusterName
+    buildSpec:
+        steps:
+        # TODO: just guessing how helm works
```

Comment:
remove this todo

Reply:
lol but it's still true! XD

okay i'll remove it ;)